lib/rules/indent.js

```diff
@@ -1011,6 +1011,11 @@ module.exports = {
 
             VariableDeclarator(node) {
                 if (node.init) {
+                    const operator = sourceCode.getTokenBefore(node.init);
```

Comment:
I think this will work incorrectly if `node.init` is parenthesized. Using `sourceCode.getTokenBefore(node.init, astUtils.isNotOpeningParenToken)` should resolve that.